### PR TITLE
Add step to wait for deployments deletions

### PR DESCRIPTION
**Pull request by jprzychodzen**
Without this step there is non-failling error

Additionally, fix missing names.

Tested manually.

/cc @prameshj

/assing @jkaniuk

### CODE REVIEW

File: clusterloader2/testing/l4ilb/config.yaml [hunk 1]
```diff
@@ -98,7 +99,8 @@ steps:
       objectTemplatePath: dep.yaml
       templateFillMap:
         NumReplicas: {{$SMALL_BACKEND_SIZE}}
-- measurements:
+- name: Waiting for creation to be completed
```
Comment (tosi3k) [inline clusterloader2/testing/l4ilb/config.yaml:102]:
```suggestion
- name: Waiting for objects creation to be completed
```

File: clusterloader2/testing/l4ilb/config.yaml [hunk 2]
```diff
@@ -148,7 +150,14 @@ steps:
       objectTemplatePath: service.yaml
     - basename: small-backends-dep
       objectTemplatePath: dep.yaml
-- measurements:
+- name: Waiting for delete to be completed
```
Comment (tosi3k) [inline clusterloader2/testing/l4ilb/config.yaml:153]:
```suggestion
- name: Waiting for objects deletion to be completed
```

**Review by tosi3k — COMMENTED** [review]
/lgtm

I think that the Kubemark presubmit will fail consistently given how red our Kubemark CI jobs are now: 
image

**Comment by prameshj** [comment]
/lgtm

Thanks

**Comment by mborsz** [comment]
I think I found a reason for failing kubemark: https://github.com/kubernetes/perf-tests/issues/1828

I will take a look tomorrow, once the kubemark is fixed.

**Pull request merged**